src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --focus_color: #00acdc;
+}
+
+html {
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.main_header {
+  display: flex;
+  width: 100%;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
+  padding-inline-start: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.list__item:nth-child(n + 2) {
+  margin-left: 20px;
+}
+
+.link_list_item {
+  white-space: nowrap;
+  padding: 22.5px 0;
+  display: flex;
+  background-color: #fff;
```

Comment:
No need to specify it.

Reply:
if will remove 
`white-space: nowrap;
  padding: 22.5px 0;
  display: flex;` 
  i will have this situation
![image](https://user-images.githubusercontent.com/86828809/232860394-ba9b0061-026d-4c4d-b7b4-4255614dfb9b.png)

i wrote `white-space: nowrap` because i dont want to have this case with "laptops & computers"
![image](https://user-images.githubusercontent.com/86828809/232860688-9d26f1c8-8123-4b7a-bc08-ba9088964fa9.png)